Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Find.swift

```diff
@@ -421,6 +421,8 @@ extension PathHierarchy.DisambiguationContainer {
                 return subtree[hash]
             } else if subtree.count == 1 {
                 return subtree.values.first
+            } else if let overloadGroupNode = subtree["overloadGroup"] {
```

Comment:
(This may become irrelevant depending on the above comment) but I wonder if it would be worth making "overloadGroup" a constant somewhere so it's less of a magic value?

Reply:
I was assuming that this specific find logic would not be necessary anymore.

Reply:
Right, i think by implementing it how David suggests, this part of the diff can be removed.